pkg/controller/common/reconciler.go

```diff
@@ -30,6 +33,36 @@ func (r *BaseIngressReconciler) ReconcileIngress(ctx context.Context, ci network
 
 	exposed := ci.GetSpec().Visibility == networkingv1alpha1.IngressVisibilityExternalIP
 	if exposed {
+		// get the namespace where operator has been installed
+		ns, err := k8sutil.GetOperatorNamespace()
```

Comment:
Do we need to check the namespace of the CI first? If it's already enrolled in another SMMR, we don't want to add it to ours.

Reply:
`k8sutil.GetOperatorNamespace()` operation is not to add  namespace to `SMMR` 
but logic is to get the namespace info where `SMMR` installed so i can do get operation to get info of `SMMR` and then check whether namespace of the CI enrolled in another SMMR or not and as discussed with you currently we can return nil if its already enrolled in another SMMR.

Reply:
Returning nil changes has been updated